doc/en/fixture.rst

```diff
@@ -844,7 +844,7 @@ Let's run the tests in verbose mode and with looking at the print-output::
 
     $ pytest -v -s test_module.py
     =========================== test session starts ============================
-    platform linux -- Python 3.x.y, pytest-3.x.y, py-1.x.y, pluggy-0.x.y -- $PYTHON_PREFIX/bin/python
+    platform linux -- Python 3.x.y, pytest-3.x.y, py-1.x.y, pluggy-0.x.y -- $PYTHON_PREFIX/bin/python3.6
```

Comment:
<3 that this line changes depending on who is releasing -- we could maybe normalize this :man_shrugging:

Reply:
@asottile there is a general plan to let a bot prep that pr